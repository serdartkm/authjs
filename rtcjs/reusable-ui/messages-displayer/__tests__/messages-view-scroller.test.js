import{h} from 'preact'
import  render from 'preact-render-to-string';
import MessagesViewScroller from '../lib/messages-view-scroller'



describe('MessagesViewScroller', () => {
    it("renders correctly", () => {
        const tree = render(<MessagesViewScroller />)
        expect(tree).toMatchSnapshot();
    })
})