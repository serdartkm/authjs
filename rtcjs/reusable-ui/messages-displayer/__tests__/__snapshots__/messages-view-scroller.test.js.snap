// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagesViewScroller renders correctly 1`] = `"<div style=\\"transform: rotate(180deg); height: 85%; background-color: #edeff2; overflow: scroll; width: 100%;\\"><div style=\\"transform: rotate(180deg);\\"></div></div>"`;
